deploy/default.properties

```diff
@@ -204,3 +204,6 @@ mlcp-vmargs=-Xmx512m
 # Temporary fix to support MarkLogic 8 EA2
 #
 evaler-port=7998
+
+# folde to ignore when deploying into modules database
+ignore-folders=.*/src/(node_modules|.tmp)/.*
```

Comment:
Move this into build.sample.properties as a comment explaining how to use it. I don't think this should be the default for everyone.